app/src/main/assets/about.html

```diff
@@ -11,4 +11,8 @@ <h5>Development</h5>
 <h5>Bugs</h5>
 <p>
 You should be aware that error logs are sent from this app in the event of a crash. Personally identifiable information is not included. You can disable this in the settings menu if you like.
+</p>
+<h5>Attributions</h5>
+<p>
+This app uses Icons made by Freepik, Pixel Perfect, Good Ware, photo3idea_studio and Pixelmeetup from www.flaticon.com
```

Comment:
Freepik seems to prefer the word [designed](https://support.freepik.com/hc/en-us/articles/208976585-Attribution-How-when-and-where-). Let's use that. While we are at it, let's also add a link to Freepik.

Reply:
Added designed and the link to flaticon was added at the end of the sentence.